# no input needed to get the visibility map
---
nav_msgs/OccupancyGrid visibility_map
